ref/cmake/compiler_flags_GNU_Fortran.cmake

```diff
@@ -1,16 +1,16 @@
 ####################################################################
 # COMMON FLAGS
 ####################################################################
-set( CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -fbacktrace -ffloat-store -fcray-pointer -fno-unsafe-math-optimizations -frounding-math -ffp-contract=off")
+set( CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -g -fbacktrace -ffloat-store -fcray-pointer -fno-unsafe-math-optimizations -frounding-math -ffp-contract=off")
```

Comment:
Should `-g` be set for all builds?  I suggest we only set it for "debug" builds.

Reply:
The -g helps with debugging and does not detract from performance, so I vote for keeping it with debug and release.  With the -g and the release aborts, we get more information about which subroutine and sometimes line number caused the abort.

Reply:
In that case, please set it in the common flags and remove it from the debug and release flags so that it properly set for both without duplication.

Reply:
@LyndStringer - Nevermind.  Trying to do too many things at the same time and misread the diff.